src/svg/Painter.js

```diff
@@ -394,7 +427,7 @@ SVGPainter.prototype = {
 
     pathToDataUrl: function () {
```

Comment:
Perhaps it's better to be named `toDataUrl`? It can be a common method for all implemented painters(canvas / svg)

Also `path` is a term that being used as a type of graphic element. It may be confusing here.